examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<p>Why <code>prefetch</code>? It is not clear what is the source of speed increase: is it the number of GPUs or bigger <code>prefetch</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>